.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "npm.packageManager": "bun",
```

Comment:
This kind of has nothing to do with the rest of the PR, but I used a github codespace to work on the PR, and this is the vscode setting that is used to setup the codespace with the proper package manager.